skrifa/src/outline/glyf/hint/engine/graphics_state.rs

```diff
@@ -151,7 +151,11 @@ impl<'a> Engine<'a> {
     pub(super) fn op_spvfs(&mut self) -> OpResult {
         let y = self.value_stack.pop()? as i16 as i32;
         let x = self.value_stack.pop()? as i16 as i32;
-        let vector = math::normalize14(x, y);
+        let vector = if x == 0 && y == 0 {
```

Comment:
totally unnecessary but you could write this as, `if (x, y) == (0, 0)`

Reply:
yes, I like this better